examples/sgd/sgd.go

```diff
@@ -9,20 +9,16 @@ func main() {
 	opt := torch.NewSGDOpt(0.1, 0, 0, 0, false)
 	opt.AddParameters([]torch.Tensor{a})
 
+	torch.PrepareGC()
 	for i := 0; i < 100; i++ {
+		torch.GC()
 		b := torch.RandN(10, 100, false)
 		pre := torch.MM(b, a)
 		loss := torch.Sum(pre)
 
 		opt.ZeroGrad()
 		loss.Backward()
 		opt.Step()
-
-		loss.Close()
-		pre.Close()
-		b.Close()
 	}
-
-	opt.Close()
-	a.Close()
+	torch.FinishGC()
```

Comment:
torch.EndTraining?

Reply:
The function may be called by a predict loop, we need a more accurate name.